### ISSUE TITLE
Problem with github API

### ISSUE THREAD

**Issue by elibarzilay** [DefinitelyTyped/dt-mergebot#311]
The recent switch from the last 100 commits to just the `headRef` broke looking at `checkSuites`, which looks like it's always empty.  I will merge a reversal of 954066f (from #305).

For an example that I tried in the [GH playground](https://docs.github.com/en/free-pro-team@latest/graphql/overview/explorer):

```gql
query PR {
  repository(owner: "DefinitelyTyped", name: "DefinitelyTyped") {
    pullRequest(number: 50168) {
      headRef {
        target {
          oid
          ... on Commit {
            checkSuites(last: 100) {
              nodes { conclusion status url }}}}}
      commits(last: 100) {
        nodes {
          commit {
            oid
            checkSuites(first: 100) {
              nodes { conclusion status }}}}}
    }
  }
}
```

**Comment by elibarzilay** [comment]
@jablko, if you have any idea about fixing it, I obviously prefer the much shorter query...

**Comment by elibarzilay** [comment]
You didn't find a way to make it work with the smaller query?

**Comment by jablko** [comment]
:disappointed: Shucks. I reproduced the problem and will ask about it on https://github.community unless someone knows a better forum?

Meanwhile I prepared #312 to revert a minimum of #305 which no longer reverts cleanly.
- Reverts changes to `_response.json`
- Switches `headRef` back to `commits` in `pr-query.ts`
- Restores `getHeadCommit()` in `pr-info.ts`

Feel free to merge it or another reversion. Apologies!
